README.md

```diff
@@ -1,20 +1,29 @@
 # [Measurement Lab](http://www.measurementlab.net/) Source Code
 
-This is the source code of the Measurement Lab website built using [Jekyll](http://jekyllrb.com) and utilizing [GitHub Pages](https://pages.github.com/) to publish and host the site.
+This is the source code of the Measurement Lab website built using
```

Comment:
OK, I can revert these changes.  I started down the path of doing that and decided if we wanted to go that route no reason to back those changes out, but I can do that.